src/Compilers/VisualBasic/Test/Semantic/Semantics/GetSemanticInfoTests.vb

```diff
@@ -5985,7 +5985,7 @@ End Module
             Dim symbolInfo = model.GetSymbolInfo(node)
 
             Assert.Null(symbolInfo.Symbol)
-            Assert.Equal(CandidateReason.NotReferencable, symbolInfo.CandidateReason)
+            Assert.Equal(CandidateReason.StaticInstanceMismatch, symbolInfo.CandidateReason)
```

Comment:
I think the reason this has changed is that we have `LocationSpecificBinder` in attribute semantic model. The binder's `ContainingMember` is the symbol for `Main` method. Previously, `ContainingMember` would have been the `Program` Module.

In the following code path, we now go `Case SymbolKind.Method` since we have the symbol for `Main`.

https://github.com/dotnet/roslyn/blob/4518b4087d086319310b9dfe055906770e1eba7d/src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb#L1208-L1230